docs-source/content/faq/cannot-pull-image.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Cannot Pull Image"
+date: 2019-03-23T08:08:19-04:00
+draft: false
+---
+
+> My domain will not start and I see errors like `ImagePullBackoff` or `Cannot pull image`
+
+When you see these kinds of errors it means that Kubernetes cannot find your Docker image.
+The most common causes are: 
+
+* The `image` value is your domain resource is set incorrectly, meaning Kubernetes will be 
```

Comment:
The `image` value is your domain resource-> The `image` value in your domain resource